k8s/jobs/mediawikiUpdate.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -o pipefail
+
+# Run MediaWiki "update.php" script in a k8s job. This can be used to update a specific wiki
+# Usage: "./mediawikiUpdate.sh <wbs_domain>"
+# Example: ./mediawikiUpdate.sh mywayordatway.wikibase.dev
+
+if [[ $# -lt 3 ]]; then
```

Comment:
This doesn't seem to match the documented usage above, as it exits when there are less than 3 parameters given, when there is only 1 expected (the domain)